packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -775,6 +784,7 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 		const socket = this.socketReference;
 		assert(socket !== undefined, 0x0a2 /* "reentrancy not supported!" */);
 		this.socketReference = undefined;
+		this._metadata = undefined;
```

Comment:
Not sure why you want to reset it - I think it's fine to keep it (though there is not much difference)

Reply:
Removed it since no longer needed after emitting event on service.